### ISSUE TITLE
DISCUSSION: Playlist functionality

### ISSUE THREAD

**Issue by codicusmaximus** [Libresonic/libresonic#459]
Does anyone use the playlist functionality?  If so, in what way?  I don't know if my needs are just different from the rest of the world or what but I find using playlists at all to be way too clunky to get any real use out of them.  I would ideally want to right click a song or click a context button to the right of the song in the list to get a small popup with a search box at the top and 10 most recently added-to playlists listed to click to add to one of them or search for the one to add to.  Also maybe even just a single button to insta-add to the current/most-recently-edited playlist.

Unless there's an easier way I'm not aware of, to add a song I find while browsing around to a "Song with epic bass drops" playlist, i have to add it to the play queue, go into the play queue, check-box the song, click into the operations dropdown to select 'add to playlist', select the playlist.

**Comment by SuperNerd-James** [comment]
I know what you mean. The Add All option in an album is misleading, it adds to play queue and not play list which is what I thought it would. There's too many clicks involved.

The "Add All" should be changed to "Add All to Playlist" and there needs to be an easier way to add individual songs :)

**Comment by codicusmaximus** [comment]
Yeah, agreed.  And the play queue is a bit too volatile for working on playlists with it (one click of play all / play song instead of add will wreck your whole play queue)

**Comment by SuperNerd-James** [comment]
Agreed. I elect that the playlist functionality be kept, as I use Libresonic when DJ'ing, keeping my music on a central server over 4G broadband when out at gigs, so playlists are vital in my case.

**Comment by codicusmaximus** [comment]
Yeah I definitely think it should be kept, I was just trying to think of ways to make it more pleasant to work with.
